docs/userguide/modules/creating-service-bindings/pages/binding-options.adoc

```diff
@@ -212,7 +215,7 @@ spec:
     name: super-secret-data
 ----
 
-Or, using the spec API:
+.Example of `ServiceBinding` CR in the `servicebinding.io` API:
```

Comment:
Why is this appearing in black color? Is it because of the changes in the content? anyways, just check once and ensure all is ok here.

Reply:
This looks fine to me on my branch.